docs/requirements/requirements.md

```diff
@@ -0,0 +1,112 @@
+# Hardware requirements of TwPM module
+
+[tcg_client_v13r27]:<https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientTPMInterfaceSpecification_TIS__1-3_27_03212013.pdf>
+[tcg_client_PTP_v1p5r14]:<https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf>
+
+## Acronyms
+
+Legend:
+
+* PTP – Platform TPM Profile
+* CRB – Command Response Buffer interface
+* DDWG – Device Driver’s Writers Guide
+* Certification PP – Certification Protection Profile
+* TIS – TPM Interface Specification
+
+## Requirements
+
+### Power supply
+
+#### Power supply voltage
+
+* TPM MUST support a supply voltage of 1.8V or 3.3V.
+* TPM MAY support supply and I/O voltages of both 1.8V and 3.3V.
+* TPM MAY support other supply voltages.
+
+Source: [tcg_client_v13r27] l.1373-1375
+
+#### Power supply max current
+
+Maximum power supply taken by TPM MUST be 250mA.
+
+Source: [tcg_client_PTP_v1p5r14], Table 60, VDD row
+
+### SPI Interface
+
+#### Frequency
+
+Default clock frequency for PC Client platforms is defined to be 24MHz.
+Clock frequency signal, minimum, is 10MHz.
+
+Source: [tcg_client_v13r27] Chapter 6.4.1, line 1331
+
+### Voltage levels
+
+1. TPM MUST support a supply and I/O voltage of 1.8V or 3.3V
+1. TPM MAY support supply and I/O voltages of both 1.8 and 3.3V.
+1. TPM MAY support other supply and I/O voltages.
+
+Source: [tcg_client_v13r27] l.1373-1375
+
+#### Power sequence
+
+* TPM_Init (LRESET#/SPI_RST#) signal connected to platform CPU Reset signal
+* TPM main power pins (3V) must be connected such that the TPM is powered
+  during ACPI states S0-S2 and may be powered in S3-S5
+* Optional implementation VBAT and/or 3VSB pins
```

Comment:
This is a requirement for the PC motherboard, it doesn't affect the choice for a hardware platform for TPM.